hieradata/common.yaml

```diff
@@ -79,6 +79,13 @@ backup::offsite::jobs:
     hour: 8,
     minute: 13,
     # No encryption because of size and sensitivity
+  'govuk-cdn-logs':
```

Comment:
On second thoughts probably not, since this will copy any directories from the logs-cdn-1 machine including the original path.